### PR TITLE
Release 0.14.0 announcement

### PR DESCRIPTION
**Pull request by lbulej**
Initial draft. File name will be updated to reflect the actual release date.

### CODE REVIEW

**Comment by lbulej** [comment]
There is also a [draft release](https://github.com/renaissance-benchmarks/renaissance/releases) with pointers to pull requests in the main project.

File: _posts/2022-01-xx-renaissance-0-14-0.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - Updated/commonized several secondary dependencies
+
+
+### Standalone mode
+
+Standalone mode means executing a benchmark without the help of the launcher
+in the main Renaissance bundle. The benchmark harness is still used, but both
+the harness and benchmark code are loaded using a single class loader. This
+is useful in situations where using multiple class loaders makes the runtime
+too convoluted, e.g., when benchmarking AOT-compiled code.
```
Comment (farquet) [inline _posts/2022-01-xx-renaissance-0-14-0.md]: I would finish the sentence with "with GraalVM Native Image" to make it clear. Even though it's the only such runtime ATM.

File: _posts/2022-01-xx-renaissance-0-14-0.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+author: Lubomír Bulej
+---
+
+We are pleased to announce a new release of the Renaissance benchmark suite.
+This release contains a mix of benchmark dependency updates and some internal
+changes. The most visible change is an update of Apache Spark to version 3.2.0,
+which enabled migrating the `apache-spark` benchmarks from Scala 2.12 to Scala
+2.13. In addition, Scala 2.13 benchmarks were updated to use Scala 2.13.8. The
+internal changes make it easier to run individual benchmarks in what we call
+_standalone mode_, which is desired in some use cases.
```
Comment (farquet) [inline _posts/2022-01-xx-renaissance-0-14-0.md]: "which is desired for GraalVM Native Image support"

**Review by farquet — APPROVED** [review]
LGTM!

File: _posts/2022-01-xx-renaissance-0-14-0.md (hunk 1 above)
Comment (lbulej) [reply _posts/2022-01-xx-renaissance-0-14-0.md]: I did not know if it's OK to be specific :-) Fine with me.

Comment (lbulej) [reply _posts/2022-01-xx-renaissance-0-14-0.md]: Done.

File: _posts/2022-01-xx-renaissance-0-14-0.md (hunk 2 above)
Comment (lbulej) [reply _posts/2022-01-xx-renaissance-0-14-0.md]: Done.

**Review by axel22 — APPROVED** [review]
LGTM

**Comment by ceresek** [comment]
Sometimes it might be useful to also mention what dependency version we update from, rather than just to (e.g. Spark in the report).

**Comment by lbulej** [comment]
> Sometimes it might be useful to also mention what dependency version we update from, rather than just to (e.g. Spark in the report).

It's in the details? Or did you mean something else?

**Comment by ceresek** [comment]
Ahh, found it, thanks :) was just looking on the GitHub links first ...

**Pull request merged**